lambda-testkit/src/main/scala/feral/lambda/testkit/TestContext.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda
+package testkit
+
+import scala.concurrent.duration.FiniteDuration
+
+object TestContext {
+  def apply[F[_]](
```

Comment:
I thought a lot about how to fix this.

I didn't want to make the `Context` constructor public in the core lambda module, because then we wouldn't be able to add new fields to it backwards-compatibly even when AWS can.

OTOH, if the public constructor is in the testkit, should a new required field be added, we can just use a dummy value as a default. This is something I wouldn't want to do in lambda core, but seems reasonable for testing :)